### PR TITLE
added modal confirmation window when deleting repositories 

### PR DESCRIPTION
**Pull request by maragh**
added confirmation window when deleting repositories.

### CODE REVIEW

**Comment by participator** [comment]
I received this error after rebuilding my image.  
Screenshot 2024-04-15 at 8 47 13 AM

**Comment by participator** [comment]
It works now.  There is an issue where the data refreshes before the repo deletes.  The causes the delete repo to appear in the refreshed data.

File: src/index.html [hunk 1]
```diff
@@ -41,9 +41,27 @@ <h1>Github Repository Control</h1>
...
+                <button type="button" class="btn btn-danger btn-sm my-2 text-uppercase" data-bs-toggle="modal" data-bs-target="#confirm_action_modal">delete repositories</button>
             </div>
         </div>
+            <!-- Modal -->
+            <div class="modal fade" id="confirm_action_modal" data-bs-backdrop="static" data-bs-keyboard="false" tabindex="-1" aria-labelledby="confirm_action_modal_label" aria-hidden="true">
+                <div class="modal-dialog">
+                <div class="modal-content">
+                    <div class="modal-header">
+                    <h1 class="modal-title fs-5" id="confirm_action_modal_label">Confirm Action</h1>
+                    <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
```
Comment (participator) [inline src/index.html]: Indenting these would make it easier to see they are inside of div.modal-header

File: src/index.html [hunk 2]
```diff
@@ -41,9 +41,27 @@ <h1>Github Repository Control</h1>
...
+                    </div>
+                    <div class="modal-body">
+                    Are you sure you want to delete the selected repositories?
```
Comment (participator) [inline src/index.html]: Indenting this would make it easier to see it is inside of div.modal-body

File: src/index.html [hunk 3]
```diff
@@ -41,9 +41,27 @@ <h1>Github Repository Control</h1>
...
+                    </div>
+                    <div class="modal-footer">
+                    <button type="button" class="btn btn-secondary" data-bs-dismiss="modal">Close</button>
+                    <button type="button" class="btn btn-primary" v-on:click="delete_repository">Confirm</button>
```
Comment (participator) [inline src/index.html]: Indenting these would make it easier to see it is inside of div.modal-footer

**Review by participator — APPROVED** [review]

**Comment by maragh** [comment]
> It works now. There is an issue where the data refreshes before the repo deletes. The causes the delete repo to appear in the refreshed data.

This is a bug that needs to be worked on separately. I will create a issue for this.

File: src/index.html (hunk 2 above)
Comment (maragh) [reply src/index.html]: I will format HTML

**Pull request merged**